src/v/crypto/include/crypto/ossl_context_service.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2024 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
```

Comment:
What stuff is going to be RCL vs BSL?

Reply:
That... is an interesting question.  Because FIPS mode is definitely an enterprise feature, but the use of OpenSSL is not.  I guess I"d argue this is BSL since it's available in the main product regardless of FIPS or not?

Reply:
Fair I wonder if it's worth while to have an implementation that is "fips mode" that is RCL, but if that's just passing a flag to openssl, I guess there isn't anything interesting there...